draft-ietf-quic-qpack.md

```diff
@@ -1551,6 +1551,169 @@ the smallest number of bytes.
 | 97    | x-frame-options                  | deny                                                        |
 | 98    | x-frame-options                  | sameorigin                                                  |
 
+
+# Encoding and Decoding Examples
+
+The following examples represent a series of exchanges between an encoder and a
+decoder.  The exchanges are designed to exercise most QPACK instructions, and
+highlight potentially common patterns and their impact on dynamic table state.
+The encoder sends three encoded field sections containing one field line each,
+as wells as two speculative inserts that are not referenced.
+
+The state of the encoder's dynamic table is shown, along with its
+current size.  Each entry is shown with the Absolute Index of the entry (Abs),
+the current number of outstanding encoded field sections with references to that
+entry (Ref), along with the name and value.  Entries above the 'acknowledged'
+line have been acknowledged by the decoder.
+
+## Literal Field Line With Name Reference
+
+The encoder sends an encoded field section containing a literal representation
+of a field with a static name reference.
+
+~~~
+Data                | Interpretation
+                              | Encoder's Dynamic Table
+
+Stream: 0
+0000                | Required Insert Count = 0, Base = 0
+510b 2f69 6e64 6578 | Literal Field Line with Name Reference
+2e68 746d 6c        |  (:path=/index.html)
```

Comment:
It might help if you include the numeric values included in each of these descriptions.  In this case, it's (static table 1).